acestep/api/http/release_task_models_test.py

```
@@ -18,20 +18,38 @@ def test_generate_music_request_preserves_legacy_defaults(self):
+        self.assertEqual(1, req.source_track_index)
+        self.assertAlmostEqual(0.3, req.source_latent_mix_ratio)
 
     def test_audio_code_string_and_cover_noise_strength_are_accepted(self):
         """Model should accept user-supplied audio_code_string and cover_noise_strength."""
 
-        req = GenerateMusicRequest(audio_code_string="<|audio_code_1|>", cover_noise_strength=0.75)
+        req = GenerateMusicRequest(
+            audio_code_string="<|audio_code_1|>",
+            cover_noise_strength=0.75,
+            repaint_mode="most natural",
+            source_session_dir="/tmp/source-session",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/source-session"

(S108)